amazon/ionbenchmark/ion_benchmark_cli.py

```diff
@@ -284,7 +294,15 @@ def _run_benchmarks(specs: list, report_fields, output_file):
         result_stats = report_stats(benchmark_spec, result, report_fields)
         report.append(result_stats)
 
-    print(tabulate(report, tablefmt='fancy_grid', headers='keys'))
+        # TODO: Add some option to dump or otherwise expose the raw sample data. For now, you can
+        # uncomment the following lines to get the raw results as CSV that can be copy/pasted into a spreadsheet.
+        #
+        # printable_key = benchmark_spec.get_name().replace(" ", "").replace(",", "-")
+        # for _x in [printable_key, *result.timings]:
+        #     print(_x, end=",")
+        # print("")
+
+    print(tabulate(report, tablefmt='pipe', headers='keys', floatfmt='.2f'))
```

Comment:
Using the new table format, you can copy/paste directly from the terminal to e.g. a github comment, and this is what you will see:
| name                                                  |   time_mean(ns) |   time_error(ns) |   ops/s_mean |   ops/s_error |
|:------------------------------------------------------|----------------:|-----------------:|-------------:|--------------:|
| (cbor,dumps,cat.cbor)                                 |          621.56 |             6.53 |   1609325.62 |      16351.13 |
| (cbor2,dumps,cat.cbor)                                |         2559.77 |            20.53 |    390726.48 |       3086.54 |
| (json,dumps,cat.json)                                 |         1619.62 |             6.14 |    617453.26 |       2341.06 |
| (ujson,dumps,cat.json)                                |          698.16 |             3.01 |   1432404.24 |       6223.26 |
| (ion_text,dumps,cat.ion)                              |         9420.41 |            84.96 |    106175.15 |        938.70 |
| (ion_binary,dumps,cat.ion)                            |        10463.80 |            59.24 |     95575.76 |        541.19 |
| Ion Binary, pure Python impl                          |       263795.71 |          1908.30 |      3791.34 |         27.22 |
| Ion Text, pure Python impl                            |       107178.88 |          1369.13 |      9334.16 |        115.81 |
| (self_describing_protobuf,dumps,cat.sd_protobuf_data) |         3974.29 |            55.22 |    251742.58 |       3371.29 |
| (protobuf,dumps,cat.protobuf_data)                    |          285.56 |             2.89 |   3502871.19 |      34420.39 |


Note that this suffers from [false precision](https://en.wikipedia.org/wiki/False_precision) (just like JMH reports do). The number formatting needs to be fixed to show a specific number of significant figures—possibly dependent on the error value for the results.